packages/manager/modules/advices/src/advices/controller.js

```diff
@@ -30,6 +39,10 @@ export default class AdvicesCtrl {
       });
   }
 
+  fetchAllAdvices() {
+    return [this.loadAdvices(), this.loadRecommenderSystemAdvices()];
+  }
```

Comment:
**nitpick**
Why not directly call these promises in `$q.all`?

Reply:
To have one dedicated location to load advices.